sympy/matrices/reductions.py

```diff
@@ -314,6 +318,47 @@ def _rref(M, iszerofunc=_iszero, simplify=False, pivots=True,
     >>> rref_pivots
     (0, 1)
 
+    If pivots is ``False`` just the row-reduced matrix is returned.
+
+    >>> m.rref(pivots=False)
+    Matrix([
+    [1, 0],
+    [0, 1]])
+
+    If different iszerofunc is used such as ``lambda x: abs(x)<6e-15``.

```

Comment:
This isn't a sentence. What happens in this case? We need to use words so that a user can understand. The example doesn't explain it.